### ISSUE TITLE
Amazon Cloud Drive large Backup fails with RequestTimeout

### ISSUE THREAD

**Issue by Tr0bs0r** [duplicati/duplicati#2289]
I have:
- [x] searched open and closed issues for duplicates

----------------------------------------

### Version info

**Duplicati Version:** 2.0.1.43
**Operating System:** Debian 7.11
**Backend:** Amazon Cloud Drive

### Bug description
When starting a large backup (minimum 400GB) to ACD it fails after transferring data for lots of hours. Smaller Backups (up to 50GB) run without problems. Duplicati shows "GetRequestStream Error" with the logs mentioned below.

Thank you for your time and help!!

### Steps to reproduce
- Start large Backup
- Backup first runs without problems
- Backup stops with the error "GetRequestStream timed out"

**Actual result:** Backup fails with GetRequestStream Timeout
**Expected result:** Backup should complete successfully without any problems

### debug log
System.Net.WebException: GetRequestStream timed out ---> System.Net.WebException: Aborted.
  at System.Net.HttpWebRequest.EndGetRequestStream (System.IAsyncResult asyncResult) [0x00043] in <bd46d4d4f7964dfa9beea098499ab597>:0 
  at Duplicati.Library.Utility.AsyncHttpRequest+AsyncWrapper.OnAsync (System.IAsyncResult r) [0x0000b] in <17dedabb0e524ef092e16e684772cd8c>:0 
   --- End of inner exception stack trace ---
  at Duplicati.Library.Main.Operation.BackupHandler.HandleFilesystemEntry (Duplicati.Library.Snapshots.ISnapshotService snapshot, Duplicati.Library.Main.BackendManager backend, System.String path, System.IO.FileAttributes attributes) [0x00011] in <5de75bdc70324cd296ed7b6669d483fd>:0 
  at Duplicati.Library.Main.Operation.BackupHandler.RunMainOperation (Duplicati.Library.Snapshots.ISnapshotService snapshot, Duplicati.Library.Main.BackendManager backend) [0x001b5] in <5de75bdc70324cd296ed7b6669d483fd>:0 
  at Duplicati.Library.Main.Operation.BackupHandler.Run (System.String[] sources, Duplicati.Library.Utility.IFilter filter) [0x005a6] in <5de75bdc70324cd296ed7b6669d483fd>:0

**Comment by kenkendk** [comment]
I think this is possibly the same as #2129 ?
How many files are created and what size do you have for the volumes?

**Comment by Tr0bs0r** [comment]
Thanks for your reply!!
In the past weeks 10.000s of files were successfully created in ACD before the Backup failed. Today and yesterday I've recreated the Backup Set to test again with the higher retrial limit. But now, whether or not I use the higher limit, only up to 20 files are created in ACD. I've tried it with the default 50MB, but also with 100,150,200 & 250 MB volumes.
So I think setting the retrial limit higher doesn't solve my problem.
